link-rest/src/main/java/com/nhl/link/rest/meta/LrAttribute.java

```diff
@@ -9,12 +10,20 @@
  */
 public interface LrAttribute {
 
+	/**
+	 * @since 1.12
+     */
 	String getName();
 
 	/**
 	 * @since 1.24
 	 */
 	Class<?> getType();
 
+	/**
+	 * @since 1.12
+     */
 	ASTPath getPathExp();
+
+	Object extractValue(JsonNode node); // TODO: move this to a separate place?
```

Comment:
Probably this can/should be moved to some shared "mapping" service, much similar to IRelationshipMapper. Something like:
```java
class JsonValueConverterMapper {
    JsonValueConverter getConverter(LrAttribute attribute);
    JsonValueConverter getConverter(LrRelationship relationship);
}
```
The trick would be to update this shared cache from the entity compiler, which performs the actual construction of converters (e.g. see CayenneEntityCompiler)